tox.ini

```diff
@@ -258,7 +258,7 @@ commands =
     -i --rm \
     -v {toxinidir}:/io \
     -e ANSIBLE_PYLIBSSH_CYTHON_TRACING \
-    ghcr.io/ansible/pylibssh-manylinux{env:MANYLINUX_VERSION_TAG}_{env:MANYLINUX_ARCH_TAG}:libssh-v{env:LIBSSH_VERSION:0.9.6} \
+    ghcr.io/ansible/pylibssh-manylinux{env:MANYLINUX_VERSION_TAG}_{env:MANYLINUX_ARCH_TAG}:libssh-v{env:LIBSSH_VERSION:0.11.1} \
```

Comment:
@Jakuje the new place where this is defined is https://github.com/ansible/pylibssh/blob/80bccfe/pyproject.toml#L164-L167. Take it into account when rebasing.

Although, the CI will be failing since there are some unsolved issues with the base containers that I haven't had a chance to troubleshoot properly. Especially, with non-x86 arches (my local QEMU setup is broken).

Reply:
Yes. I saw that already, but I did not see a lot of value in pushing it right now. But I can do that to keep it up to date.